imports/plugins/included/ui-search/lib/components/searchModal.js

```diff
@@ -145,6 +145,132 @@ class SearchModal extends Component {
     );
   }
 
+  constructor(props) {
```

Comment:
Lines 148 to 273 repeats already defined methods in the class. Please delete them and include any change you might have added to the methods in the existing class methods. This [gives a better view of the repetition](https://github.com/andela/annie-hall-rc/blob/0589238e256f0206a61ef84e7788b98ccf31cc4a/imports/plugins/included/ui-search/lib/components/searchModal.js#L148-L272). Those methods exist in line 146 backwards

Reply:
fixed